cli/src/commands/operation/diff.rs

```diff
@@ -486,6 +522,14 @@ fn get_parent_changes(
     }
 }
 
+fn get_ordered_wc_ref_targets(repo: &ReadonlyRepo) -> Vec<(String, RefTarget)> {
+    repo.view()
+        .wc_commit_ids() // Returns an ordered map
+        .iter()
+        .map(|(k, v)| (k.as_str().to_string(), RefTarget::normal(v.clone())))
```

Comment:
nit: it's probably better to not stringify the `WorkspaceId` here. Typed value is nicer.

Reply:
Sure, NP.